javascript/ql/src/semmle/javascript/frameworks/Bundling.qll

```diff
@@ -222,10 +222,10 @@ predicate isMultiLicenseBundle(TopLevel tl) {
     )
   ) > 1
   or
-  // case: ordinary block comments with "@license" lines
+  // case: ordinary block comments lines that start with a license
   count(BlockComment head |
     head.getTopLevel() = tl and
-    head.getLine(_).regexpMatch("(?i) *\\* @license .*")
+    head.getLine(_).regexpMatch("(?i)[\\s*]*(@license\\b.*|The [a-z0-9-]+ License (\\([a-z0-9-]+\\))?\\s*)")
```

Comment:
What about `.` in license names? Also, I suppose some licenses have multi-word names?

Reply:
I am deliberately very conservative here to avoid flagging actual comments like:

```
/**
 * The user needs to accept the licence.
 */
function hasAcceptedLicense(){...}
```
Unlike the disjunct above, the comment block for this disjunct does not have the special exclamation point marker, so there is a bigger risk of FPs.

The change is motivated by "The MIT License (MIT)", but I thought the hardcoding for "MIT" was a bit too restrictive.